ENH: More specific tear-down details, recognizing recent events (#361)
TheAxonLab/hcph-sops: docs/data-collection/tear-down.md

### ET arm, ET cables, infrared mirror and stimuli screen

- [ ] Unplug the two cables (signal and power) connected to the ET arm.

> - [ ] Roll the ET's cables and put them in the cupboard inside the Scanning room.

Reviewer: Specifying two may avoid forgetting one cable (who knows at this point)
```suggestion
- [ ] Roll the two ET cables and put them in the cupboard inside the Scanning room.
```

    !!! danger "The infrared mirror MUST be manipulated with clean gloves at all times."

- [ ] Take the projector's screen off and store it in its designated shelf.

### Gas mask/cannula and tubing, RB and tubing, ECG cables

- [ ] Roll the ECG cable after checking the leads are bagged.
- [ ] Bag the ECG cable once rolled.
- [ ] Detach the RB tube from the RB inlet.
- [ ] Roll the RB tube and the GA tube and store them bagged in the cupboard.

### Readying the scanner for the next session

!!! important "Reset all scanner settings in the Scanning Room to defaults"

    Every setting altered for the experiment MUST be put back to its original status at the end of the experiment (e.g., position of the bed, coil, emergency button, padding elements, etc.)

- [ ] Remove used blankets and bed-sheets <mark>**ONE-BY-ONE**</mark>:
    - [ ] extend them <mark>**ONE-BY-ONE**</mark> to let any forgotten items fall on the floor before you fold it; and
    - [ ] dispose of them <mark>**ONE-BY-ONE**</mark> in the adequate bin (soiled linen bag if they are fabric and trash if they are disposable).

    ??? danger "Make sure you do not accidentally dispose of valuable items or instrumentation"

>         At the end of the scanning session everyone is tired and the likelihood of a misshapen is large because the attention levels are at at their lowest.

Reviewer: ```suggestion
        At the end of the scanning session everyone is tired and the likelihood of a misshapen is large because the attention levels are at their lowest.
```

- [ ] Take the ET arm outside to the Control Room and place it in a stable place.
- [ ] Take the infrared outside to the Control Room and store it in the ET/fMRI box.
- [ ] Take the plexiglass panel outside to the control room.

> - [ ] Take the RB outside the scanning room and box it in the ET/fMRI box.

Reviewer: Good idea to store it there

- [ ] Exit and close the external door.

## Clearing up the Control Room

### Finalize the *boxing* of the ET elements

- [ ] Put the lens cover.

> - [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its pouch.

Reviewer 2: ```suggestion
- [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it, put back its cover and put the lens back into its pouch.
```
